samples/project/Get-Project-Information.yaml

```diff
@@ -0,0 +1,80 @@
+id: ciz79iwgk0000456w9wch1l0m
+gist: ''
+source: PROJECT
+author: Microsoft
+name: Get information about the project
+description: >-
+  Shows how to use project level fields to find things like the project's start
+  date
+script:
+  content: |
+
+
+    $('#run').click(getProjectInformation);
+
+      // Get the specified fields for the active project.
+        function getProjectInformation() {
+            var fields =
+                [Office.ProjectProjectFields.Start, Office.ProjectProjectFields.Finish, Office.ProjectProjectFields.GUID];
+            var fieldValues = ['Start: ', ' Finish: ', ' GUID: '];
+            var index = 0; 
+            getField();
+
+            // Get each field, and then display the field values in the add-in.
+            function getField() {
+                if (index == fields.length) {
+                    var output = '';
+                    for (var i = 0; i < fieldValues.length; i++) {
+                        output += fieldValues[i];
+                    }
+                    console.log(output);
+                }
+                else {
+                    Office.context.document.getProjectFieldAsync(
+                        fields[index],
+                        function (result) {
+
+                            // If the call is successful, get the field value and then get the next field.
+                            if (result.status === Office.AsyncResultStatus.Succeeded) {
+                                fieldValues[index] += result.value.fieldValue;
+                                getField(index++);
+                            }
+                            else {
+                                onError(result.error);
+                            }
+                        }
+                    );
+                }
+            }
+        }
+
+        function onError(error) {
+            console.log(error);
+        }
+  language: typescript
+style:
+  content: /* Your style goes here */
+  language: css
+template:
+  content: |-
+    <button id="run" class="ms-Button">
```

Comment:
Add short description: "This snippet demonstrates how to get project information such as..."